### PR TITLE
Added readme docs for the ssh_config_file option.

### PR DESCRIPTION
**Pull request by Vasu1105**
Signed-off-by: Vasu1105 <vasundhara.jagdale@chef.io>

## Description

Readme Doc update for #709 , #713
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by Vasu1105** [comment]
@clintoncwolfe Could you please review this and merge it?

**Review by clintoncwolfe — APPROVED** [review]
Technically correct but needs some wordsmithing to read smoothly, I think.

File: README.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ require 'train'
 train = Train.create('ssh', host: '1.2.3.4', port: 22, user: 'root')
 ```
 
+SSH transport has `ssh_config_file` option to set the ssh config file path. This is by default set to `true` to read the values from the default ssh config file path for e.g. `~/.ssh/config`, `/etc/ssh_config`, `/etc/ssh/ssh_config`. Precedence is given to the options set through the arguments to the create method.
```
Comment (IanMadd) [inline README.md:68]:
```suggestion
SSH transport has an `ssh_config_file` option to set the SSH config file path. This is set by default to `true` to read the values from the default SSH config file path. For example, `~/.ssh/config`, `/etc/ssh_config`, `/etc/ssh/ssh_config`. Precedence is given to the options set through the arguments to the create method.
```

**Comment by Vasu1105** [comment]
@clintoncwolfe I did the required changes after the docs team review. I think this PR is good to merge.

**Pull request merged**